### PR TITLE
support for audit logs

### PR DESCRIPTION
**Pull request by zhu733756**
/kind feature

### CODE REVIEW

**Comment by wanjunlei** [comment]
You can add a doc to Introduce how to use `auditd`, and collect audit logs with fluentbit-operator.

File: README.md [hunk 1]
```diff
@@ -103,6 +104,21 @@ green open ks-logstash-log-2020.04.26 uwQuoO90TwyigqYRW7MDYQ 1 1  99937 0  31.2m
...
+```shell
+kubectl apply -f manifests/setup
+kubectl apply -f manifests/logging-stack/auditd
+```
+
+Within a couple of minutes, you should observe an index available:
+
+```shell
+$ curl localhost:9200/_cat/indices
+green open ks-auditd-logs-2021.04.06 QeI-k_LoQZ2h1z23F3XiHg  5 1 404879 0 298.4mb 149.2mb
```
Comment (benjaminhuo) [inline README.md:119]: still `ks-auditd-logs-`

**Review by benjaminhuo — CHANGES_REQUESTED** [review]
`ks-auditd-logs-` should be modified too

File: README.md (hunk 1 above)
Comment (zhu733756) [reply README.md]: fixed

**Review by benjaminhuo — APPROVED** [review]
/lgtm
@wanjunlei pls review this PR, and possibly we need to release FB operator v0.4.1

**Review by wanjunlei — APPROVED** [review]

**Pull request merged**